ext/curl/php_curl.h

```diff
@@ -38,6 +38,8 @@ extern zend_module_entry curl_module_entry;
 PHP_CURL_API extern zend_class_entry *curl_ce;
 PHP_CURL_API extern zend_class_entry *curl_share_ce;
 PHP_CURL_API extern zend_class_entry *curl_multi_ce;
+PHP_CURL_API extern zend_class_entry *curl_CURLUrl_ce;
```

Comment:
Nit: it would be slightly more fortunate to follow the camel_case format of the other CE variables, like: `curl_url_ce `

Reply:
There is already inconsistency we also have 
```
PHP_CURL_API extern zend_class_entry *curl_CURLFile_class;
PHP_CURL_API extern zend_class_entry *curl_CURLStringFile_class;
```
So either choice is fine with me but I think this one is a little bit better since we see the classname in it.